recipes/pexports/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+if(NOT WIN32)
+    message(FATAL_ERROR "pexports only supports Windows")
+endif()
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
```

Comment:
why so complex CMakeLists.txt for test_package?
did you borrow it somewhere?

Reply:
No, it's homebrew.
It's just creating a dll -> running pexports on the dll -> running dlltool/lib to create an import library -> create an exe using the import library.

It's do-able outside cmake by just sequentially executing the commands, but I can re-use some cmake variables here.